### DISCUSSION TITLE
https://klp1.com/parts/printer-parts/

### DISCUSSION THREAD

**Discussion by giscus** [blakadder/klp1#16] [Announcements]
# https://klp1.com/parts/printer-parts/

Printer parts for Kingroon KLP1

https://klp1.com/parts/printer-parts/

**Comment by wFirbolg** [comment]
Hi everyone, 

I've been wanting to add a 24v fan extractor to evacuate fumes when printing ASA/ABS. Could I use the PA3 or other available connectors on the motherboard to control it?

Many thanks

**Reply by jstillwell** [reply]
Yes, you can. There are several available ports. I have done some mods and made a [blog post](https://graybush.io/blog/kingroon-klp1-230-upgrades/) that may be helpful.

Good luck!
